hayeon/programmers-2/피보나치-수.js

```diff
@@ -0,0 +1,16 @@
+function solution(n) {
+  let fiboNum = [0, 1];
+
+  for (let i = 2; i <= n; i++) {
+    fiboNum[i] = (fiboNum[i - 1] + fiboNum[i - 2]) % 1234567;
+  }
+  return fiboNum[n];
+}
```

Comment:
```suggestion
function solution(n) {
  let fiboNumList = [0, 1];

  for (let i = 2; i <= n; i++) {
    const sum = (fiboNumList[i - 1] + fiboNumList[i - 2]) % 1234567;
      
    fiboNumList.push(sum);
  }
  return fiboNumList[n];
}
```
- 깔끔한 풀이 잘봤씁니다 🙇‍♂️
- ⭐️ fiboNum보다는 배열이기떄문에 `복수`로 작성하던가 List라는 네이밍을 주는 것을 추천드립니다.
- ⭐️ fiboNum[i]에다 할당하더라도 배열은 const로 선언이 가능합니다.
- 물론 위에 수정 코드처럼 push로 빼주셔도 상관없습니다~

Reply:
- 헉 배열이 맞네요!!!! 피드백 감사합니다!!
- push 로 빼는게 더 깔끔해 보이네요.. 참고히겠습니당!!👍🏻